packages/maskbook/src/plugins/ITO/SNSAdaptor/NftAirdropCard.tsx

```diff
@@ -0,0 +1,337 @@
+        borderRadius: 4,
+    },
+    imgWrapper: {
+        display: 'inline-block',
+        width: 65,
+        height: 85,
+        marginRight: theme.spacing(1),
+    },
+    gallery: {
+        width: '100%',
+        marginBottom: theme.spacing(0.5),
+        overflowX: 'scroll',
+        whiteSpace: 'nowrap',
+    },
+    actionButton: {
+        height: 40,
+        width: 130,
+        background: 'rgba(255, 255, 255, 0.2)',
+        color: '#fff',
+        '&:hover': {
+            background: 'rgba(255, 255, 255, 0.4)',
+        },
+        '&.Mui-disabled': { color: '#fff', opacity: 0.6, background: 'rgba(255, 255, 255, 0.2)' },
+    },
+    connectWallet: {
+        width: 200,
+        minHeight: 40,
+        whiteSpace: 'nowrap',
+    },
+    disabledButton: {
+        color: '#fff',
+    },
+    address: {
+        width: 340,
+        color: '#fff',
+        '& fieldset': {
+            borderColor: '#fff !important',
+        },
+    },
+    addressInput: {
+        color: '#fff',
+        padding: 13,
+    },
+    chainBoundary: {
+        display: 'none',
+    },
+    loading: {
+        marginLeft: theme.spacing(1),
+        color: 'rgba(255, 255, 255, 0.4)',
+    },
+    snackbarContent: {
+        color: '#fff',
+        display: 'flex',
+        width: 250,
+        marginLeft: theme.spacing(0.5),
+        justifyContent: 'space-between',
+        alignItems: 'center',
+    },
+    snackbarTipContent: {
+        display: 'flex',
+        flexDirection: 'column',
+    },
+    snackbarTip: {
+        fontSize: 13,
+        display: 'flex',
+        alignItems: 'end',
+    },
+    snackbarIcon: {
+        marginLeft: theme.spacing(0.3),
+        height: 17,
+        width: 17,
+        cursor: 'pointer',
+    },
+    whiteText: {
+        color: '#fff',
+    },
+    closeIcon: {
+        cursor: 'pointer',
+    },
+}))
+
+interface NftAirdropCardProps {
+    campaignInfo: CampaignInfo
+}
+
+export function NftAirdropCard(props: NftAirdropCardProps) {
+    const { t } = useI18N()
+    const [checkAddress, setCheckAddress] = useState('')
+    const now = Date.now()
+    const { campaignInfo } = props
+
+    const [spaceStationClaimableCount, spaceStationAccountClaimableCallback, spaceStationAccountClaimableLoading] =
+        useSpaceStationClaimableTokenCountCallback()
+    const account = useAccount()
+    const currentChainId = useChainId()
+    const { value: claimInfo, loading } = useSpaceStationClaimable(account)
+    const claimable = Boolean(claimInfo?.claimable) && currentChainId === ChainId.Mumbai
+    const classes = useStyles()
+    const [claimState, claimCallback] = useSpaceStationContractClaimCallback(campaignInfo!)
+    const { enqueueSnackbar, closeSnackbar } = useSnackbar()
+    const snackbarOptions = {
+        preventDuplicate: true,
+        anchorOrigin: {
+            vertical: 'top',
+            horizontal: 'center',
+        },
+    }
+
+    useEffect(() => setCheckAddress(''), [account])
```

Comment:
Do you forget to add `chainId`?

```suggestion
    useEffect(() => setCheckAddress(''), [account, chainId])
```